src/styles/pages/contactUs.scss

```diff
@@ -220,7 +220,10 @@
     }
 
     .address-col {
-      transform: translateX(20%);
+      @include mobile {
```

Comment:
You can just remove the `transform: translateX`, we don't need to make the address part center neither on PC nor mobile.